storage/client.go

```diff
@@ -432,6 +433,17 @@ func (c *Client) AddToUserAgent(extension string) error {
 	return fmt.Errorf("Extension was empty, User Agent stayed as %s", c.userAgent)
 }
 
+// AddAdditionalHeaders adds additional standard headers
+func (c *Client) AddAdditionalHeaders(headers map[string]string) error {
```

Comment:
Since no error is ever returned can this return type be removed?

Reply:
Done